File: explorer-be-main/src/HospitalLibrary/DTOs/RegistrationDTO.cs
using System;
using System.Collections.Generic;
using HospitalLibrary.Core.Model;
using HospitalLibrary.Core.Model.Enum;

namespace HospitalLibrary.DTOs
{
    public class RegistrationDto
    {
        public string Password { get; set; }
        public string Email { get; set; }
        public string Username { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public List<string> Interests { get; set; }
    }
}

File: explorer-fe-main/src/app/modules/pages/register/RegistrationDto.ts
export class RegistrationDto {
  firstName: string;
  lastName: string;
  username: string;
  email: string;
  password: string;
  interests: string[];

  constructor(
    firstName: string = '',
    lastName: string = '',
    username: string = '',
    email: string = '',
    password: string = '',
    interests: string[] = []
  ) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.username = username;
    this.email = email;
    this.password = password;
    this.interests = interests;
  }
}
